compiler/optimizer/OMRTransformUtil.cpp

```diff
@@ -394,3 +394,56 @@ OMR::TransformUtil::transformCallNodeToPassThrough(TR::Optimization* opt, TR::No
    node->removeAllChildren();
    node = TR::Node::recreateWithoutProperties(node, TR::PassThrough, 1, child);
    }
+
+void
+OMR::TransformUtil::createConditionalAlternatePath(TR::Compilation* comp,
+                                                   TR::TreeTop *ifTree,
+                                                   TR::TreeTop *thenTree,
+                                                   TR::Block* elseBlock,
+                                                   TR::Block* mergeBlock,
+                                                   TR::CFG *cfg,
```

Comment:
Out of curiosity why do we need to provide a CFG here? Is it possible for a CFG to be "rogue" and live outside the `comp` object? Or to have multiple CFGs? I'm wondering if this parameter could be removed and just extract the CFG from the compilation?

Reply:
Good point. I was using the existing Block API as a template when writing this one, didn't think of why we need a CFG as a parameter.

Looking at the usage of `OMR::Block::createConditionalBlocksBeforeTree`, we use it in openj9's ilgen where we pass the inlined method's CFG, which is different than the top level method's CFG.

Reply:
Thanks, did not think about the inlined method CFG case. No further comments from me.